src/inputs/internal/ComboBoxBase.tsx

```diff
@@ -110,24 +108,33 @@ export function ComboBoxBase<O, V extends Value>(props: ComboBoxBaseProps<O, V>)
     [propOptionMenuLabel, unsetLabel, getOptionLabel],
   );
 
+  // Call `initializeOptions` to prepend the `unset` option if the `unsetLabel` was provided.
+  const options = useMemo(
```

Comment:
Hardly anything needed the "maybe-ness" of `maybeOptions`, so just converted this into a consistent `O[]`.